app/services/financial_year_close.rb

```diff
@@ -11,16 +11,60 @@ def initialize(year, to_close_on, options = {})
     @options = options
   end
 
+  def benchmark(message)
+    start = Time.now
+    moment = '[' + format('%.1f', start - @start).rjust(6).red + '] '
+    Rails.logger.info(moment + message.yellow + '...')
+    # puts moment + message.yellow + '...'
+    yield
+    stop = Time.now
+    # puts moment + message.yellow + " (done in #{sprintf('%.2f', stop - start).green}s)"
+    Rails.logger.info(moment + message.yellow + " (done in #{format('%.2f', stop - start).green}s)")
+  end
+
+  def say(message)
+    now = Time.now
+    moment = '[' + format('%.1f', now - @start).rjust(6).red + ' '
+    # puts @counts.stringify_keys.to_yaml.cyan if @counts
+
+    moment << if @previous_now
+                format('%.2f', now - @previous_now).rjust(6).green
+              else
+                '—' * 6
+              end
+    moment << '] '
+
+    t = moment + message.yellow
+    # puts t
+    Rails.logger.info(t)
+    now
+  end
+
+  def log(message)
+    @previous_now = say(message)
+  end
+
+  def cinc(name, _increment = 1)
```

Comment:
If we're done testing the FinancialYearClose, I'd like that we remove all the testing code that was set up by @burisu. Unless we couple it to a proper reporting system, it has no place in production.